src/Our.Umbraco.Ditto/ComponentModel/TypeConverters/MultiNodeTreePickerConverter.cs

```diff
@@ -58,7 +60,9 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
                 var multiNodeTreePicker = Enumerable.Empty<T>();
 
                 int n;
-                var nodeIds = s
+                var nodeIds = XmlHelper.CouldItBeXml(s)
+                    ? uQuery.GetXmlIds(s)
```

Comment:
Thinking about my uQuery concern, (since I know it will be removed in Umbraco v8.0), I tried to find an alternative API within the Umbraco, but no luck!

I reckon that we should include the `GetXmlIds()` as an internal helper method (within Ditto). Would `ConverterHelper` be an appropriate place to have it?